homeassistant/components/vodafone_station/icons.json

```diff
@@ -0,0 +1,44 @@
+{
+  "entity": {
+    "device_tracker": {
+      "device_tracker": {
+        "default": "mdi:lan-disconnect",
+        "state": {
+          "home": "mdi:lan-connect"
+        }
+      }
+    },
```

Comment:
I think this is the default nowadays for device tracker and there is no need for per-integration implementation.

Reply:
I though it was for `person` not for `device_tracker`